client/src/main/java/io/getstream/chat/android/client/models/Attachment.kt

```diff
@@ -30,8 +32,25 @@ data class Attachment(
     var name: String? = null,
     var fallback: String? = null,
 
+    @IgnoreSerialisation
+    @IgnoreDeserialisation
+    var upload: File? = null,
+
+    @IgnoreSerialisation
+    @IgnoreDeserialisation
+    var uploadState: UploadState? = null,
+
     @IgnoreSerialisation
     @IgnoreDeserialisation
     override var extraData: MutableMap<String, Any> = mutableMapOf()
 
-) : CustomObject
+) : CustomObject {
+
+    companion object {
```

Comment:
it's not necessary to keep class inside companion object, it rather complicates stuff